### ISSUE TITLE
Calendar won't see the previous year's notes

### ISSUE THREAD

**Issue by denizonm** [liamcain/obsidian-calendar-plugin#343]
### Describe the bug

After the start of the new year, the calendar has stopped recognizing the previous year's notes. The naming scheme is the same, because whenever I try to create a file that does exist, it tells me it is unable to create said file.

My naming scheme is kind of unorthodox, it is "WW-E...gg" if that makes any difference.

I am using this in conjunction with the Periodic Notes plugin.

### Steps to reproduce

1. Go to calendar
2. See that the note associations have disappeared

### Expected behavior

Calender shows the previous year's notes as well.

#### OS

Linux Mint 

#### Obsidian Version

v1.5.3

#### plugin versions
Calendar: 1.5.10
Periodic Notes: 0.0.17

Thank you!

**Comment by DesertDwarf** [comment]
I'm curious, can you give a real example of a note's name based on the naming scheme?

**Comment by denizonm** [comment]
Thank you for your response.

Today would be 2-1..24

(second week of the year)-(first day of the week)..(last 2 digits of the year)

I was wondering if it's the ambiguity of the year since 23 can stand for 2023 or 3023 technically, but that doesn't seem to be the problem.

**Comment by DesertDwarf** [comment]
That's what I was wondering when I first read your question. So, you never see the month in that date format, correct? I'm not sure the Calendar plugin can even understand what date you're trying to reference since (to its way of thinking) you're missing a key element: the month.

I don't see anything in the configuration of the current beta that allows you to tell it what your date format is except with a Locale, and those are all standard language-location choices. I believe you'd need to customize the underlying program directly to handle your unique case.

Edited to Add: I did some snooping in the newer version (the beta) and it may be possible he's handling your kind of date format now. Have you tried the beta?

You should be able to switch back and forth as needed:

1. Go into your vault folder's `.osbidian` folder
2. Then `plugins` folder
3. Find the `calendar` folder
4. Copy or move it to somewhere else
5. Download the three files found here: https://github.com/liamcain/obsidian-calendar-plugin/releases/tag/2.0.0-beta.2 - Specifically, main.js, manifest.json, and syles.css
6. Put them in a `calendar` folder inside the `.osbsidian/plugins` folder. You'll have to create `calendar` if you moved the old calendar folder in step 4.

If you don't like the beta version, delete all files in the `.obsidian/plugins/calendar` folder and copy the files inside the calendar file you copied/moved in step 4.

I've been using the beta version with only one issue, which has behaved OK. (The one issue is that if I have the "Words" sources set to 0, then any day I write stuff in my journal, the day number disappears in the calendar month view. It might be a theming issue, but I've been using the default theme. If I have a number (so I get dots) or I mark "Words" as hidden, I still see the day numbers.)

**Comment by denizonm** [comment]
Thank you for your help. 

The beta didn't seem to do it for me either.
I have been trying a whole bunch of stuff, including going to the beta and changing the scheme to everything I could think of.

It seems the problem was the `gg` at the end. It seems it does show the year, but it confuses the plugin.

In the [documentation](https://momentjs.com/docs/#/displaying/format/) it is under the locale week section, so maybe that's why? It's probably just a very obscure way of doing it. In any case, changing the naming scheme to WW-E..YY has solved all my problems.

I've also gone back and checked if it the new naming scheme makes it work within the non-beta version, and it does.

Thanks again for your help.

tl;dr: using `gg` to denote year works in every case except for looking at notes from previous years.
